ush/ioda/bufr2ioda/bufr2ioda_conventional_prepbufr_ps.py

```diff
@@ -39,6 +39,33 @@ def Mask_typ_for_var(typ, var):
     return typ_var
 
 
+def Compute_obsSubType(typ, t29, sid):
+
+    obssubtype = ma.array(np.full(typ.shape[0], 0))
+    for i in range(len(typ)):
+        newtyp = typ[i]
+        if ((typ[i] == 180) or (typ[i] == 280)) and (t29[i] == 562):
+            newval = 0
+            try:
+                newval = int(sid[i][-3:])
+            except Exception:
```

Comment:
is there a specific type of exception we should be capturing here?

Reply:
Probably a valueError. Let me test real quick